prac_05/word_occurrences.py

```diff
@@ -20,7 +20,9 @@ def main():
 
 def print_word_count(word_to_count):
```

Comment:
You don't need to use a def for this code and if you were to use it, you need a def main() above this.